blueprints/showcase-plugin-with-media/blueprint.json

```
@@ -0,0 +1,102 @@
+{
+	"$schema": "https://playground.wordpress.net/blueprint-schema.json",
+	"meta": {
+		"title": "Showcase plugin",
```

@Lovor01 the metadata will be used to display the Blueprint title and description, so it would be worth using something more descriptive like you did in the readme. 

```suggestion
		"title": "Showcase custom plugin from own server with media files and custom theme",
```